rosdep/python.yaml

```diff
@@ -2978,6 +2978,22 @@ python-numpydoc:
   gentoo: [dev-python/numpydoc]
   opensuse: [python-numpydoc]
   ubuntu: [python-numpydoc]
+python-nuscenes-devkit-pip:
```

Comment:
this is a python2 entry, do you want to add this package only for python2?

if you meant to add the python3 key, it should be `python3-nuscenes-devkit-pip`

Reply:
@ivanpauno Fixed at https://github.com/ros/rosdistro/pull/31455/commits/dfa022734d8c3b56a12b6d9233db687fabf5b3c8.